modules/administration/pages/content-lifecycle-examples.adoc

```diff
@@ -184,6 +184,7 @@ The new environment contains all the kernel patches up to the version you instal
 NOTE: All kernel patches with higher kernel versions are removed.
 Live patching kernels remain available as long as they are not the first in a series.
 
+NOTE: This procedure can be automated using a filter template. For more information on how to apply a live patching filter template, see xref:administration:content-lifecycle.adoc#filter-templates[].
```

Comment:
Maybe, we can turn this into a formal note (it's new, thus it may deserve more visibility):
```suggestion
[NOTE]
====
This procedure can be automated using a filter template.
For more information on how to apply a live patching filter template, see xref:administration:content-lifecycle.adoc#filter-templates[].
====
```